cmd/lnmuxd/server.go

```diff
@@ -23,24 +24,30 @@ const (
 	// subscriberQueueSize is the length of the rpc subscriber queue. If the
 	// queue becomes full, the client is disconnected.
 	subscriberQueueSize = 100
+
+	// defaultPaginationSize is the default number of invoices returned by the pagination
+	defaultPaginationSize = 100
```

Comment:
Maybe we should do no default, to force the caller to think about it.

Reply:
Interesting, Usually (in the wider world), there is a default, but It  is a good idea to "empower" the user

Reply:
With lnd, we've had quite some instances of users not using the api correctly because of defaults. The most notable default was the payment fee limit at 100% of the payment amount.